components/ItemCard/ItemCard.js

```diff
@@ -0,0 +1,66 @@
+// Import external resources and hooks
+import styled from "styled-components";
+import { useState } from "react";
+
+// Import of styled components
+import * as Styled from "@/components/ItemCard/ItemCard.styled.js";
+
+// Import of SVG Icon Components
+import * as Icon from "@/components/Icons";
+
+export function ItemCard({ product, setCartTotal, cartTotal }) {
+  const [counter, setCounter] = useState(0);
+
+  function increaseCounter(oldCounter) {
+    const newCounter = oldCounter === 99 ? oldCounter : oldCounter + 1;
+    setCounter(newCounter);
+  }
+
+  function decreaseCounter(oldCounter) {
+    const newCounter = oldCounter === 0 ? oldCounter : oldCounter - 1;
+    setCounter(newCounter);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    if (counter > 0) {
+      setCartTotal(cartTotal + counter);
+      setCounter(0);
+    }
+  }
+
+  return (
+    <li>
+      <Styled.ArticleCard>
+        <span>
+          <Styled.Thumbnail
+            src={product.thumbnail}
+            alt="no img"
+            width={120}
+            height={150}
+          />
+        </span>
+        <span>
+          <h2>{product.title}</h2>
+          <p>{product.price}€</p>
+          <form onSubmit={handleSubmit}>
```

Comment:
You don't need the form here. It's semantic not correct 'cause it's not really a form.

Reply:
Yeah you are I removed the input field.

1. It was not part of my userstory
2. Input number caused me some trouble ( Already go some ideas to solve it later, if really need it)

- [x] <form> → <div> (styled will replaced with a grid layout later)
- [x] replaced handleSubmit → handleClick